docs/deployment/manage-projects.md

```diff
@@ -0,0 +1,81 @@
+---
+slug: manage-projects
+title: Manage projects
+hide_title: true
+description: View projects, detailed logs and information for any scan.
+tags:
+    - Deployment
+    - Semgrep AppSec Platform
+---
+
+# Manage projects
+
+View, sort, and tag your projects through the **Projects** page. A **project** is any repository that you have scanned with Semgrep AppSec Platform through either CI or Semgrep Managed Scans, and includes that repository's findings and metadata.
+
+Refer to this page to **manage** and **troubleshoot** thousands of repositories by identifying scan issues or scans with a high finding count.
+
+Additionally, the Semgrep API enables you to filter tags for additional workflows and integrations into your own systems. Create tags based on engineering or department teams, external-facing or internal codebases, and so on. See [Tags](/semgrep-appsec-platform/tags) for more information.
```

Comment:
This paragraph feels a bit out of place, so maybe move it to the end of the section / just above the Sort projects section.